k8s/namespaces/probate/probate-caveats-fe/demo.yaml

```diff
@@ -6,7 +6,7 @@ spec:
   releaseName: probate-caveats-fe
   values:
     nodejs:
-      image: hmctspublic.azurecr.io/probate/caveats-fe:pr-457-dee5c7a-20220405162732 #{"$imagepolicy": "flux-system:demo-probate-caveats-fe"}
+      image: hmctspublic.azurecr.io/probate/caveats-fe:latest #{"$imagepolicy": "flux-system:demo-probate-caveats-fe"}
```

Comment:
This change should be like - 

image: hmctspublic.azurecr.io/probate/caveats-fe:pr-457-dee5c7a-20220405162732 #{"$imagepolicy": "flux-system:probate-caveats-fe"}